pkg/omf/functions/cli/omf.cli.update.fish

```diff
@@ -1,12 +1,25 @@
 function omf.cli.update
-  if omf.core.update
-    echo (omf::em)"Oh My Fish is up to date."(omf::off)
+  set -l update_core
+  set -l packages $argv
+  if test (count $packages) -gt 0
```

Comment:
I adopted the first suggestion for it's explicitness and readability.
For the same reasons, I'd like to keep the more explicit version here.

Thanks for the CR!